problems_1/problems_1.tex

```diff
@@ -0,0 +1,264 @@
+\documentclass[12pt,twoside]{article}
+\setlength{\textwidth}{166mm}   % ширина текста
+\setlength{\textheight}{232mm}  % высота текста на странице
+\setlength{\topmargin}{-10mm}   % верхнее поле (+ 1 дюйм)
+\setlength{\headsep}{5mm}       % отступ после колонтитула
+\oddsidemargin=3mm              % левое поле для нечетной страницы (+1 дюйм)
+\evensidemargin=3mm             % левое поле для четной страницы (+1 дюйм)
+\setlength{\baselineskip}{18pt} %
+\usepackage[utf8]{inputenc}
+\usepackage[russian]{babel}
+\usepackage{amsfonts,amssymb,amsmath}
+\usepackage{epsfig}
+\usepackage{mathrsfs}
+\usepackage{mathabx}
+\usepackage{eufrak}
+\usepackage{xcolor}
+
+\renewcommand\le{\leqslant}
+\renewcommand\ge{\geqslant}
+
+\newcommand{\ruText}[1]{{\scriptsize \textcolor{darkgray}{#1} \par}}
+
+% Lines below are to avoid word breaks.
+\tolerance=1
+\emergencystretch=\maxdimen
+\hyphenpenalty=10000
+\hbadness=10000
+
+\renewenvironment{itemize}
+{\begin{list}
+             {\labelitemi}%                     Old parameters:
+             {\setlength{\labelwidth}{1.3em}%        1em
+              \setlength{\labelsep}{0.7em}%          0.7em
+              \setlength{\itemindent}{0em}%          0em
+              \setlength{\listparindent}{3em}%       3em
+              \setlength{\leftmargin}{2em}%          3em !
+              \setlength{\rightmargin}{0em}%         0em
+              \setlength{\parsep}{0ex}%              0ex
+              \setlength{\topsep}{0.5ex}%            2ex !
+              \setlength{\itemsep}{1ex}%             0ex
+             }%
+}
+{\end{list}}%
+
+
+\begin{document}
+\pagestyle{empty}
+
+\subsection*{
+  {\normalsize 1. Выказванні. Лагічныя аперацыі над выказваннямі. Формулы.} \\
+  \indent \ruText{ Высказывания. Логические операции над высказываниями. Формулы.}
+}
+
+\begin{itemize}
+
+\item[1.]
+
+Вылучыце ўмову і вынік тэарэмы, сфармулюйце яе з дапамогай звязкі <<Калі $\ldots$, то $\ldots$>>: \\
+\ruText{ Выделив условие и заключение теоремы, сформулируйте её посредством связки <<Если $\ldots$, то $\ldots$>>: }
+\begin{itemize}
+
+\item[(а)] Для таго, каб функцыя была дыферэнцавальнай ў некаторай кропцы, неабходна, каб яна была непарыўнай у гэтай кропцы; \\
```

Comment:
I think, redefining enumerate is better, but it's not necessary

Reply:
You mean to avoid hardcoding Cyrillic letters?

Reply:
Cyrillic letters and numbers for \item

Reply:
https://tex.stackexchange.com/questions/139462/controlling-enumerate-nesting-level

Reply:
ok I'll try to do that

Reply:
I did not see you link. I've done it in a slightly different way.